pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -1648,6 +1740,84 @@ func (gce *GCECloud) ListClusters() ([]string, error) {
 	return result, nil
 }
 
+func (gce *GCECloud) ListClusters() ([]string, error) {
```

Comment:
See above.  It looks like we might be able to delete this entire thing, or just return nil (like AWS and all other cloud providers seem to do).